src/client/pages/LoginApp/components/LoginForm.tsx

```diff
@@ -122,7 +122,6 @@ export const LoginForm: React.FC<LoginFormProps> = ({ onSubmit }) => {
                 id="ssn"
                 type="text"
                 inputMode="numeric"
-                pattern="[0-9]*"
```

Comment:
Why change this?

Reply:
We already have inputMode numeric on this which only allows characters between 0 and 9 so this is not needed.

Reply:
Seems like Safari (desktop) doesn't support "inputMode" => so perhaps we need to keep it for that reason. But could be good to add that as a comment.

https://caniuse.com/input-inputmode

Reply:
oh Safari 😦 . great catch 👍 .